test/integration-test/tests/load.rs

```diff
@@ -94,7 +95,10 @@ macro_rules! assert_loaded_and_linked {
 macro_rules! assert_loaded {
     ($name:literal, $loaded:expr) => {
         for i in 0..(MAX_RETRIES + 1) {
-            let state = loaded_programs().any(|prog| prog.unwrap().name() == $name.as_bytes());
+            let state = loaded_programs()
+                .filter(|prog| prog.is_ok())
```

Comment:
instead of filter->unwrap can you use filter_map(prog.ok()) and add a comment?

Reply:
Done